### PR TITLE
GEODE-7641: clean up compiler warnings

### PR DESCRIPTION
**Pull request by jinmeiliao**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by jinmeiliao** [comment]
@jmelchio

File: geode-core/build.gradle [hunk 1]
```diff
@@ -177,6 +177,7 @@ dependencies {
   compileOnly files("${System.getProperty('java.home')}/../lib/tools.jar")
 
   //Find bugs is used in multiple places in the code to suppress findbugs warnings
+  compileOnly("io.swagger:swagger-annotations")
```
Comment (metatype) [inline geode-core/build.gradle]: Why is this needed in geode-core?  AFAICT, the swagger annotations are not used in geode-core.  There's also a runtime dependency declaration.  Which one is correct?  Note that the runtime declaration is also marked as optional.

Comment (jinmeiliao) [reply geode-core/build.gradle]: just testing this out. Will revert this.
